### ISSUE TITLE
Update Bluemix Deployment Instructions To Use Different Mongo Service

### ISSUE THREAD

**Issue by ryanjbaxter** [meanjs/mean#1029]
The MongoLab service in Bluemix will be going away at the end of November.  We need to update the deployment instructions as well as the auto deployment button to use a different Mongo service.

**Comment by ryanjbaxter** [comment]
This is dependent on #1025 being fixed first

**Comment by lirantal** [comment]
@ryanjbaxter  Cool, can you keep an eye out for this and follow with a PR?

**Comment by ryanjbaxter** [comment]
@lirantal yes waiting on #1025 and then I will update everything and submit a pull request

**Comment by lirantal** [comment]
@ryanjbaxter you're good to go, #1025 is merged.

**Comment by ryanjbaxter** [comment]
@lirantal see #1039

**Comment by ryanjbaxter** [comment]
I will also need to update the gh-pages instructions in a separate pull request i believe

**Comment by lirantal** [comment]
Sure. See the comments there from @ilanbiala
